mpsl/CHANGELOG.rst

```diff
@@ -9,6 +9,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Master branch
+*************
+
+Added
+=====
+* Added a new signal to MPSL timeslot ``MPSL_TIMESLOT_SIGNAL_OVERSTAYED``.
+  This signal is given to the application when a timeslot session is closed too late (DRGN-14677).
+
+Bugfixes
+========
+
+* Fixed an issue where the low frequency clock was configured incorrectly when low frequency source was either low or full swing signal (DRGN-15064).
```

Comment:
@greg-fer How can we rephrase this section to match our DRGN-15064 known issue description:
https://github.com/nrfconnect/sdk-nrf/pull/3402

Reply:
Yes - it will not be present in v1.4.1 because we are also porting the fix to the v1.4 branch:
https://github.com/nrfconnect/sdk-nrfxlib/pull/349